UnityProject/Assets/Scripts/Core/Editor/Inspector/InterfaceGUI.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Systems.ObjectConnection;
+using Mirror;
+using UnityEditor;
+using UnityEngine;
+using System.Collections.Generic;
+
+namespace CustomInspectors
+{
+	[ExecuteInEditMode]
+	public class InterfaceGUI : NetworkBehaviour
```

Comment:
name is way too generic for its very specific use which is rendering the interface for multitool masters

Reply:
and door buttons! and a third one later

Reply:
door buttons are as well using the multitool subscription system, aren't they?